### PR TITLE
net-dns/openresolv-3.11.0: version bump

### PR DESCRIPTION
**Pull request by acerisara**
Bump to 3.11.0, add support to notify avahi-daemon and mdnsd of resolv.conf changes.

### CODE REVIEW

**Comment by charIes17** [comment]
> Bump to 3.11.0, add support to notify avahi-daemon and mdnsd of resolv.conf changes.

Why don't you put the bug link?
Bug: https://bugs.gentoo.org/738100

File: net-dns/openresolv/openresolv-3.11.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+DESCRIPTION="A framework for managing DNS information"
+HOMEPAGE="https://roy.marples.name/projects/openresolv"
+SRC_URI="https://roy.marples.name/downloads/${PN}/${P}.tar.xz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE="selinux"
+
+DEPEND="!net-dns/resolvconf-gentoo
```
Comment (charIes17) [inline net-dns/openresolv/openresolv-3.11.0.ebuild:15]: See https://qa-reports.gentoo.org/output/gentoo-ci/output.html#net-dns%2fopenresolv

**Comment by charIes17** [comment]
Do not forget to get the bug linked:
> In order to force reassignment and/or bug reference scan, please append `[please reassign]` to the pull request title.

**Comment by thesamesam** [comment]
Thanks!

**Pull request closed without merging**